components/cam/src/cpl/atm_comp_mct.F90

```diff
@@ -956,4 +1055,272 @@ subroutine atm_write_srfrest_mct( x2a_a, a2x_a, &
 
   end subroutine atm_write_srfrest_mct
 
+  subroutine read_surface_emis(ncols,lats2,lons2,mn,surface_emis)
+  !  This subroutine is made by UM team on Dec.15, 2019
+  !  This subroutine is to read surface emissivity from dataset
+
+      use netcdf
+  
+      use time_manager, only: get_curr_date
+      use ppgrid
+
+      use error_messages, only : handle_ncerr
+      use parrrtm,        only: nbndlw
+      use cam_logfile,     only: iulog
+
+      implicit none
+      integer :: ncid, status, latid,lonid,bandid,timeid
+      character(256) filename
+
+      integer :: ntime, nlat, nlon, nband,i,mn
+      real, allocatable :: band_emissivity(:)
+      character(len = nf90_max_name) :: RecordDimName
+      integer :: lat_varID,emis_varID, lon_varID
+      real, allocatable:: lat(:), lon(:)
+      integer ::start(4),count(4)
+      integer :: ncols,j
+      integer ::ilats, ilons
+      real(r8) :: lats2(ncols)           ! array of chunk latitudes
+      real(r8) :: lons2(ncols)           ! array of chunk longitude
+      real :: surface_emis(ncols, nbndlw), minvalue
+
+      filename = "/global/cscratch1/sd/xianwen/data/surface_emissivity_1x1_RRTMG_53deg.nc"
```

Comment:
We cannot had stuff hard-coded like that.

Reply:
Totally agreed, @golaz . All these will be taken care of soon. Note that the input of this climatology surface emissivity data also need to be reworked.